src/generator/AutoRest.Go.Tests/src/tests/acceptancetests/integergrouptest/body-integer_test.go

```diff
@@ -82,24 +77,24 @@ func (s *IntegerSuite) TestPutMin64(c *chk.C) {
 	c.Assert(err, chk.IsNil)
 }
 
-// func (s *IntegerSuite) TestGetUnixTime(c *chk.C) {
-// 	unixTime, err := intClient.GetUnixTime()
-// 	c.Assert(err, chk.IsNil)
-// 	c.Assert(unixTime, chk.Equals, time.Date(2016, time.April, 13, 0, 0, 0, 0, time.UTC).Unix())
-// }
-
-// func (s *IntegerSuite) TestPutUnixTimeDate(c *chk.C) {
-// 	_, err := intClient.PutUnixTimeDate(time.Date(2016, time.April, 13, 0, 0, 0, 0, time.UTC).Unix())
-// 	c.Assert(err, chk.IsNil)
-// }
-
-// func (s *IntegerSuite) TestGetInvalidUnixTime(c *chk.C) {
-// 	_, err := intClient.GetInvalidUnixTime()
-// 	c.Assert(err, chk.NotNil)
-// }
-
-// func (s *IntegerSuite) TestGetNullUnixTime(c *chk.C) {
-// 	res, err := intClient.GetNullUnixTime()
-// 	c.Assert(err, chk.IsNil)
-// 	c.Assert(res.Value, chk.IsNil)
-// }
+func (s *IntegerSuite) TestGetUnixTime(c *chk.C) {
+	res, err := intClient.GetUnixTime()
+	c.Assert(err, chk.IsNil)
+	c.Assert(*res.Value, chk.Equals, date.UnixTime(time.Date(2016, time.April, 13, 0, 0, 0, 0, time.UTC)))
```

Comment:
I'm not sure this really tests much, but I suppose it would alert us if we ever changed UnixTime to be something other than an alias for time.